parcellab/microservice/values.yaml

```diff
@@ -71,6 +71,49 @@ externalSecret:
   #   - extract:
   #       key: app-key-in-secrets-manager
 
+# Enable Argo Rollouts Deployments Strategies
+# See https://argo-rollouts.readthedocs.io/en/stable/
+rolloutDeployment:
+  enabled: false
+  ## BlueGreen deployment example
+  # blueGreen:
+  #   activeService: service-name
+  #   previewService: service-name-preview
+
+  ## Canary deployment example
+  # canary:
+  #   maxSurge: "25%"
+  #   maxUnavailable: 0
+  #   steps:
+  #   - setWeight: 10
+  #   - pause:
+  #       duration: 1h
+  #   - setWeight: 20
+  #   - pause: {}
+
+  ## BlueGreen deployment with Analysis template.
+  # blueGreen:
+  #   metrics:
+  #   - name: smoke-tests
+  #     interval: 5m
+  #     successCondition: result <= 0.01
+  #     failureLimit: 3
+  #     provider:
+  #       datadog:
+  #         apiVersion: v2
+  #         interval: 5m
+  #         query: |
+  #           sum:requests.error.rate{service:app}
+  #   spec:
+  #     activeService: service-name
```

Comment:
Would it make sense to generate this name out of the deployment etc ?

Reply:
that name is generated from the deployment already yes

Reply:
that is already generated from the deployment and is integrated in the template. I will remove it from the examples.